releases/sdk5.md

```diff
@@ -45,6 +45,12 @@ No changes are required for apps using the modular framework.
 
 ## Release notes ##
 
+### SDK 5.2 (October 11, 2018) ###
+- Modular Framework: Improvement in design by fixing the text height to the image height.
```

Comment:
This statement is too vague. "image" here is the app icon, and the change is in the app title.